### ISSUE TITLE
handleSlashCommand isCommand check error

### ISSUE THREAD

**Issue by chudders1231** [larsrickert/discord-player-plus#4]
When using the example provided, passing the interaction through to handleSlashCommand throws an error reporting .isCommand as not a function.

`[ERROR] 20:06:36 TypeError: interaction.isCommand is not a function`

**Comment by larsrickert** [comment]
Hey, which version of `discord-player-plus` and `discordjs` are you using?

**Comment by chudders1231** [comment]
Very sorry, i wrote that real quick and forgot to include any of that info.

discord-player-plus: 1.1.0
discord.js: 14.0.3

I have only tried the /help command. I quickly setup the example to mess around and instantly hit a road block.
Unsure if you need any additional info.

**Comment by larsrickert** [comment]
Thanks for providing the information! I reproduced the steps from the [Getting Started](https://discordplayerplus.lars-rickert.de/guide/getting-started.html) and [Pre-build Slash Commands](https://discordplayerplus.lars-rickert.de/guide/commands.html) documentation and it works without any error.

The missing function (`interaction.isCommand is not a function`) was implemented in `discord.js` v13 and `discord-player-plus` < `1.1.0`. Can try to follow the below steps to check if that fixes the issue?`

1. Verify that you are using at least `discord-player-plus` version `^1.1.0` and `discord.js` version `^14.0.3`
2. Delete your `node_modules` folder and `package-lock.json`
3. Run `npm install`
4. Check if the error still exists
